coaster/sqlalchemy/mixins.py

```diff
@@ -218,18 +219,23 @@ class UrlForMixin(object):
     """
     Provides a :meth:`url_for` method used by BaseMixin-derived classes
     """
-    #: Mapping of {action: (endpoint, {param: attr})}, where attr is a string or tuple of strings.
-    #: This particular dictionary is only used as a fallback. Each subclass will get its own dictionary.
-    url_for_endpoints = {}
+    #: Mapping of {app: {action: (endpoint, {param: attr})}}, where attr is a string or tuple of strings.
+    #: The same action can point to different endpoints in different apps. The app may also be None as fallback.
```

Comment:
Seems like `None` is also for Blueprints?

Reply:
Yes. Blueprint handling is undefined behaviour at the moment. We have no means to tell which apps the Blueprint will be registered with. It could be one, both or neither. The safest assumption is that the Blueprint will be used in all apps.